### PR TITLE
Document System.IO APIs

### PR DESCRIPTION
**Pull request by carlossanlop**

### CODE REVIEW

File: xml/System.IO/BinaryReader.xml [hunk 1]
```diff
@@ -1114,8 +1114,8 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <summary>Reads an integer 7 bits at a time.</summary>
```
Comment (pgovind) [inline xml/System.IO/BinaryReader.xml:1117]: The API is called `Read7BitEncodedInt64`. I'm wondering if the word `integer` is still right here?

File: xml/System.IO/BinaryWriter.xml [hunk 2]
```diff
@@ -2431,8 +2431,8 @@ This member is an explicit interface member implementation. It can be used only
         <Parameter Name="value" Type="System.Int64" Index="0" FrameworkAlternate="net-5.0" />
       </Parameters>
       <Docs>
-        <param name="value">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="value">The value to write.</param>
+        <summary>Writes out an integer 7 bits at a time.</summary>
```
Comment (pgovind) [inline xml/System.IO/BinaryWriter.xml:2435]: Same question here

**Review by pgovind — APPROVED** [review]
Couple comments for the area owner. Looks good otherwise

File: xml/System.IO/BinaryReader.xml (hunk 1 above)
Comment (carlossanlop) [reply xml/System.IO/BinaryReader.xml:1117]:
```suggestion
        <summary>Reads a number 7 bits at a time.</summary>
```

File: xml/System.IO/BinaryWriter.xml (hunk 2 above)
Comment (carlossanlop) [reply xml/System.IO/BinaryWriter.xml:2435]:
```suggestion
        <summary>Writes out a number 7 bits at a time.</summary>
```

**Pull request merged**